tests/unit/records/test_records_receivers.py

```diff
@@ -598,3 +600,111 @@ def test_populate_abstract_source_suggest_does_nothing_if_record_is_not_hep():
         {'source': 'foo'},
         {'source': 'bar'},
     ]
+
+
+def test_populate_affiliation_suggest():
```

Comment:
I totally agree with splitting 👍  it wasn't even easy to write and change it for me